requirements.txt

```diff
@@ -1,6 +1,6 @@
 pip>=9.0.1
 numpy>1.16
-sympy<1.6
+sympy>=1.8
```

Comment:
or <=1.8 ? or == ?
do we also support 1.5.1? 1.7?

Reply:
Only 1.8 there is some backward incompatibility with 1.5, haven't checked 1.7